### PR TITLE
Functions and test cases added

### LINKED ISSUE #16366: No function to calculate shear stress and axial stress
In `beam.py` file of `continuum_mechanics` folder there is no function to calculate shear stress and axial stress.

### PR DESCRIPTION
**Pull request by Psycho-Pirate**
#### References to other Issues or PRs

Fixes #16366

#### Brief description of what is fixed or changed
Area as an argument and area setter function was added in 2D beam class.
Functions to calculate shear stress and axial stress in 2D and 3D beams were added in beam.py
Test cases for shear and axial stress were added in test_beam.py

#### Release Notes

physics.continuum_mechanics

- Added functions to calculate shear and axial stress in beam.py

- Area can now be passed as an argument for beam class. The new API looks like this
`>>> b = Beam( length, E, I, A)`

### CODE REVIEW

**Comment by Psycho-Pirate** [comment]
I always perform bin/test and bin/doctest, how should i check for trailing whitespace errors?

**Comment by Sc0rpi0n101** [comment]
> I always perform bin/test and bin/doctest, how should i check for trailing whitespace errors?

You can run the code quality test using `./bin/test quality` and remove the trailing whitespaces using `./bin/strip_whitespace <file>`.

You should check https://github.com/sympy/sympy/wiki/Development-workflow#be-sure-that-all-tests-of-sympy-pass for further reference.

**Review by Sc0rpi0n101 — CHANGES_REQUESTED** [review]
There are multiple instances of `self.area` and `self._area`. 
You should decide which one you want to use and change all the instances to use a single identifier.

**Comment by Psycho-Pirate** [comment]
@Sc0rpi0n101 I have made the required changes. Please review.

**Comment by Psycho-Pirate** [comment]
I have made the requested changes, please review @Sc0rpi0n101

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -158,15 +170,15 @@ def variable(self):
 
         >>> from sympy.physics.continuum_mechanics.beam import Beam
         >>> from sympy import symbols
-        >>> E, I = symbols('E, I')
+        >>> E, I, A= symbols('E, I, A')
         >>> x, y, z = symbols('x, y, z')
```
Comment (Smit-create) [inline sympy/physics/continuum_mechanics/beam.py]: Please add spaces around equal to sign.

File: sympy/physics/continuum_mechanics/tests/test_beam.py [hunk 2]
```diff
@@ -141,15 +153,24 @@ def test_Beam():
...
     p = b3.load
     q = -2*SingularityFunction(x, 2, 2) + 2*SingularityFunction(x, 3, 0) + 4*SingularityFunction(x, 3, 1) + 2*SingularityFunction(x, 3, 2)
     assert p == q
 
+    p = b3.shear_force()
+    q = -2*SingularityFunction(x, 2, 3)/3 + 2*SingularityFunction(x, 3, 1) + 2*SingularityFunction(x, 3, 2) + 2*SingularityFunction(x, 3, 3)/3
+    assert p == q
+
+    p = b3.shear_stress()
+    q = -1*SingularityFunction(x, 2, 3)/3 + 1*SingularityFunction(x, 3, 1) + 1*SingularityFunction(x, 3, 2) + 1*SingularityFunction(x, 3, 3)/3
```
Comment (Smit-create) [inline sympy/physics/continuum_mechanics/tests/test_beam.py]: Please break these long lines with each line whitin 72 characters, using PEP8. Apply these to all long lines

File: sympy/physics/continuum_mechanics/beam.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ class Beam(object):
          + Piecewise(((x - 2)**4, x - 2 > 0), (0, True))/4)/(E*I)
     """
 
-    def __init__(self, length, elastic_modulus, second_moment, variable=Symbol('x'), base_char='C'):
+    def __init__(self, length, elastic_modulus, second_moment, area = Symbol('A'), variable=Symbol('x'), base_char='C'):
         """Initializes the class.
```
Comment (Smit-create) [inline sympy/physics/continuum_mechanics/beam.py]: No need to add spaces around `=` in the function arguments.

File: sympy/physics/continuum_mechanics/beam.py [hunk 4]
```diff
@@ -98,6 +98,8 @@ def __init__(self, length, elastic_modulus, second_moment, variable=Symbol('x'),
             bending axis of the beam. The second moment of area will be computed
             from the shape object internally.
 
+        area : A symbol or a value representing the cross section area of beam
+
```
Comment (Smit-create) [inline sympy/physics/continuum_mechanics/beam.py]: Correct it in the numpy doc format as:
```
area : Symbol/float 
      Represents the cross-section area of beam
```

**Review by Sc0rpi0n101 — COMMENTED** [review]
LGTM.
The tests are also passing.  
Can you add some more tests for the shear and axial stress for 3D beam?

Also, Can you squash your commits into one commit with a proper description of the changes?
You can check out https://github.com/sympy/sympy/wiki/Development-workflow#writing-commit-messages for reference

**Comment by Psycho-Pirate** [comment]
I have squashed the commits.
I was planning on creating a function to plot shear and axial stress as part of my gsoc proposal, so i was thinking about adding more examples along with their plots later.
@Sc0rpi0n101

**Comment by Psycho-Pirate** [comment]
> I have squashed the commits.
> I was planning on creating a function to plot shear and axial stress as part of my gsoc proposal, so i was thinking about adding more examples along with their plots later.

What do you suggest? @Sc0rpi0n101

**Comment by Sc0rpi0n101** [comment]
If you're planning to add more examples and tests later, then I think there is a fair amount of tests. :+1: 
Thank you for your contribution :)

**Comment by Sc0rpi0n101** [comment]
Ohh... I just realized it would've been better to have release notes for this change as the API is being changed and new functionalities are added. 

But since there's some time since the next release,  you can incliude these changes in your release notes for further changes. @Psycho-Pirate

**Comment by Psycho-Pirate** [comment]
I have updated the release notes. Thanks for reviewing. :) @Sc0rpi0n101

**Pull request merged**